lexos/helpers/general_functions.py

```diff
@@ -140,20 +97,19 @@ def merge_list(word_lists):
     return merged_list
 
 
-def load_stastic(file):
-    """
-    this method takes an ALREADY SCRUBBED chunk of file(string), and convert
-    that into a WordLists
-    (see :return for this function or see the document for 'test' function)
+def load_stastic(input_string: str) -> dict:
+    """this method takes an ALREADY SCRUBBED chunk of file(string), and
```

Comment:
cannot put change of line in the one line summary.

You can put multiline documentation after the empty line.

Please re-read the standard for documentation. Or reading other people's documentation can also be helpful